libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -480,6 +500,17 @@ bool WaveTrack::Interval::IsPlaceholder() const
    return mpClip->GetIsPlaceholder();
 }
 
+void WaveTrack::Interval::MarkChanged()
+{
+   ForEachClip([](auto &clip){ clip.MarkChanged(); });
+}
+
+void WaveTrack::Interval::Resample(int rate, BasicUI::ProgressDialog *progress)
+{
+   // TODO:  correct the denominator of the progress, when it's not null
```

Comment:
Such progress is also used in `WaveTrack::Resample` in a way that doesn't seem correct to me (not taking the number of clips into account and the progression from one clip to the next). I was tempted to add a QA item for that but I expect that our testers won't be happy and yet this wouldn't be a regression.